scraper_service/main.py

```diff
@@ -1,15 +1,20 @@
 import shutil
 from time import time
+from random import randint
+from datetime import datetime, timezone
```

Comment:
Can we remove this line as `timezone ` is not being used?

Reply:
yes let's remove it.